syft/workers/base.py

```diff
@@ -116,11 +116,11 @@ def __init__(
         self._message_router = {
             CommandMessage: self.execute_command,
             PlanCommandMessage: self.execute_plan_command,
-            ObjectMessage: self.set_obj,
+            ObjectMessage: self.handle_object_msg,
             ObjectRequestMessage: self.respond_to_obj_req,
             ForceObjectDeleteMessage: self.rm_obj,  # FIXME: there is no ObjectDeleteMessage
             IsNoneMessage: self.is_tensor_none,
-            GetShapeMessage: self.get_tensor_shape,
+            GetShapeMessage: self.handle_get_shape_message,
             SearchMessage: self.respond_to_search,
             ForceObjectDeleteMessage: self.force_rm_obj,
```

Comment:
Same: this method on object_storage doesn't expect a message

Reply:
Yeah, this is a second method that wraps the first one.

Reply:
Wait, you mean `force_rm_obj`? Yeah, that one doesn’t either. Needs a wrapper method.

Reply:
Yep, that what I meant :)